internal/jsonapi/manifest/manifest_path_freebsd.go

```diff
@@ -0,0 +1,22 @@
+package manifest
+
+var manifestPaths = &manifestPath{
+	local: map[string]string{
+		"firefox":  "~/.mozilla/native-messaging-hosts",
+		"chrome":   "~/.config/google-chrome/NativeMessagingHosts",
+		"chromium": "~/.config/chromium/NativeMessagingHosts",
+		// "brave":    "~/.config/BraveSoftware/Brave-Browser/NativeMessagingHosts",
+		// "vivaldi":  "~/.config/vivaldi/NativeMessagingHosts",
+		"iridium":  "~/.config/iridium/NativeMessagingHosts",
+		// "slimjet":  "~/.config/slimjet/NativeMessagingHosts",
```

Comment:
Why the commented out browsers?

Reply:
There is no support for those browsers in FreeBSD

Reply:
In that case please just remove them.